src/main/java/views/monthly_menu/MonthMenuV.java

```diff
@@ -0,0 +1,188 @@
+package views.monthly_menu;
+
+import entities.Category;
```

Comment:
This class too should not be importing the Category and Expense classes. If you recall from our Clean Architecture Slides (04, slide 12), Controllers should not be importing Entities, so this definitely shouldn't be happening (unless you have a good reason for it and can explain!)

Reply:
Expense and Category are basic object types we have for conveying information. Thus if we remove the imports and change the outputdata to Object[][] type in the UpdateViewUCI to accomodate Java Swing, it is not adhering to SOLID in point that its not Open to importing other libraries.
 
Of course, this is a assignment, so we should consult the Prof and/or TA about this. I will be able to change this part any time, so for now, I think we should keep it like this.

Reply:
Sounds good!